src/python/detectors/missing_seed_in_pytorch_random_number_generator/missing_seed_in_pytorch_random_number_generator.py

```diff
@@ -0,0 +1,21 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  SPDX-License-Identifier: Apache-2.0
+
+# {fact rule=missing-seed-in-pytorch-random-number-generator@v1.0 defects=1}
+def missing_seed_in_pytorch_random_number_generator_noncompliant():
```

Comment:
I'd suggest to use this compliant case https://code.amazon.com/packages/NotebookGQLRules/blobs/mainline/--/src/test/resources/PyTorchControlSourcesOfRandomness/conformant8.py, as we consider all the random sources not just from torch.